ruby/ql/integration-tests/query-suite/ruby-code-quality.qls.expected

```diff
@@ -1,3 +1,2 @@
 ql/ruby/ql/src/queries/performance/DatabaseQueryInLoop.ql
-ql/ruby/ql/src/queries/variables/DeadStoreOfLocal.ql
```

Comment:
This query has `precision medium`.

Reply:
@github/codeql-ruby could you check if you're okay with this change?

Reply:
ping @yoff .